site/docs/patterns/metric.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Metric
+layout: DetailPattern
+aliases:
+  - /salt/patterns/metric
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-metric--metric", label: "Examples" },
+      { href: "https://www.figma.com/file/TI8zR16jnisL4W7ANedtUJ/(WIP)-Metric?type=design&node-id=652-3336&mode=design&t=UpxMBoYUQ8vMxz6z-0", label: "Metric examples", internal: true },
+    ]
+---
+
+Metric is a pattern that displays an important number value prominently. Indicators or labels can be added to help provide context to the presented value.
+
+  <Diagram
+    src="/img/patterns/metric/metric.png"
+    alt="Two example metrics displaying positive and negative indicators."
+    contentPosition="top"
+    border
+  />
+
+We provide design guidance for using Salt [Icon](/salt/components/icon), [Link](/salt/components/link), [Stack Layout](/salt/components/stack-layout) and [Text](/salt/components/text) components to achieve the pattern. You can also find implementation examples in the [Storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-metric--metric).
+
+## When to use it
+
+Use the pattern to give more prominence to a key measure. Metrics are often used as building blocks in dashboards. They effectively direct users to key information that may otherwise be lost within the content, particularly when multiple numerical values are displayed.
```

Comment:
```suggestion
Use the pattern to give more prominence to a key measure. Metrics are often used as building blocks in dashboards. They effectively direct users to key information that may otherwise be lost within the content, particularly when multiple numerical values are displayed. Metrics can encompass a broad range of quantitative data, such as financial statistics, performance indicators and user engagement.
```
@yunjungyeh what do you think of these examples?